packages/@ourworldindata/grapher/src/slopeCharts/SlopeChart.tsx

```diff
@@ -1072,7 +1026,6 @@ class LabelledSlopes
         })
 
         // Order by focus/hover and size for draw order
```

Comment:
very much a nit, but:
```suggestion
        // Order by focus/hover for draw order
```